pandas/tests/groupby/test_groupby_dropna.py

```diff
@@ -162,6 +162,27 @@ def test_groupby_dropna_series_by(dropna, expected):
     tm.assert_series_equal(result, expected)
 
 
+def test_slice_groupby_then_transform():
```

Comment:
if not crazy can you parameterize on dropna (its a fixture) so we have all in the same place

Reply:
rewrote the test with `parametrize`

incidentally i ran the `dropna=True` case and found `transform` failing. I xfailed the test here and opened #35612 to track the problem

Reply:
ok saw that, okie dokie